1xx-pool-rack-id.md

```diff
@@ -0,0 +1,130 @@
+# Node Pool Rack IDs
+
+Rack-awareness in Strimzi Kafka can be achieved without broker access to cluster-level Kubernetes APIs
+by configuring a rack ID in each node pool for usage across all brokers within the pool.
+
+## Current situation
+
+In Kubernetes clusters spanning multiple availability zones, Kafka can tolerate the loss of an
+entire zone by ensuring partition replicas are spread across brokers in multiple zones.
+
+This is achieved by configuring the [`broker.rack`](https://kafka.apache.org/documentation/#brokerconfigs_broker.rack)
+property in Kafka to enable rack-aware replication assignment, where a rack is analogous to an availability zone.
+
+Strimzi currently provides rack-awareness through the usage of an init container which queries the
+Kubernetes API for the value of a specified topology label on the Kubernetes node where that
+broker is running.
+This requires access to the Kubernetes API by the broker pods and requires cluster-scoped RBAC.
+
+## Motivation
+
+For users interested in a heightened security posture, the requirements of the current rack-awareness
+implementation are prohibitive.
+Many Kubernetes cluster administrators may restrict access to cluster-scoped Kubernetes
+resources to ensure an application and the user managing it are contained within a limited set of namespaces.
+Today, Strimzi only requires access to cluster-scoped Kubernetes resources for rack-awareness and NodePort
+listener configuration.
+
+Implementing the proposed method for pool-based rack awareness removes these potentially prohibitive
+requirements while maintaining a simple deployment model.
```

Comment:
It does not remove these requirements -> it just makes it _optional_ in one of the mentioned cases.